### PR TITLE
Commvault Plugin: execute backup job

### PR DESCRIPTION
**Pull request by razi-ahmad1**

### CODE REVIEW

File: src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy [hunk 1]
```diff
@@ -70,4 +73,33 @@ class CommvaultPlugin extends Plugin {
         }
         return backupProvider
     }
+
+    def captureActiveSubclientBackup(authConfig, subclientId, clientId, backupsetId) {
```
Comment (ddeyoung) [inline src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy]: This appears to be a method that should go in the API service (CommvaultBackupUtility?).

File: src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy [hunk 2]
```diff
@@ -70,4 +73,33 @@ class CommvaultPlugin extends Plugin {
...
+        def activeJobsResults = CommvaultBackupUtility.getBackupJobs(authConfig, subclientId, [query: [clientId: clientId, backupsetId: backupsetId, jobFilter: "Backup", jobCategory: "Active"]])
+        if(activeJobsResults.success && activeJobsResults.results.size() > 0) {
+            def activeBackupJob = activeJobsResults.results.find { it.clientId == clientId && it.subclientId == subclientId && it.backupsetId == backupsetId }
+            return ServiceResponse.success([backupJobId: activeBackupJob.backupJobId])
+        }
+    }
+
+    def getBackupStatus(backupState) {
```
Comment (ddeyoung) [inline src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy]: Move this method to the `CommvaultComputeUtility` and rename the `CommvaultComputeUtility` to `CommvaultReferenceUtility`.

**Review by ddeyoung — CHANGES_REQUESTED** [review]

File: src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy (hunk 1 above)
Comment (razi-ahmad1) [reply src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy]: @ddeyoung changes has been implemented as suggested.

File: src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy (hunk 2 above)
Comment (razi-ahmad1) [reply src/main/groovy/com/morpheusdata/commvault/CommvaultPlugin.groovy]: @ddeyoung changes has been implemented as suggested.

**Review by ddeyoung — APPROVED** [review]

**Pull request merged**